### ISSUE TITLE
LMR+PVS

### ISSUE THREAD

**Issue by ChizhovVadim** [amanjpro/zahak#229]
I think, if LMR>0, research should be if score > alpha without score < beta. Or I do not understand how it works :-).

score := -e.alphaBeta(depthLeft-1-LMR, searchHeight+1, -alpha-1, -alpha)
			if score > alpha && (LMR>0||score < beta) {
...
}

https://github.com/amanjpro/zahak/blob/72ff8066e2d8348d8d3847fab62c4a6b8344240d/search/search.go#L745-L751

**Comment by amanjpro** [comment]
Yeah, I doubt the `beta` will change anything really... it might hurt too. Will check it :)
Thanks for the report :)
